libtransact/src/context/manager.rs

```diff
@@ -119,11 +112,12 @@ impl<
 
     /// Get the values associated with list of keys, from a specific Context.
     /// If a key is not found in the context, State is then checked for these keys.
+    /// Keys are returned with the associated value, if found in Context or State.
     pub fn get(
         &self,
         context_id: &ContextId,
-        keys: &[K],
-    ) -> Result<Vec<(K, Option<V>)>, ContextManagerError> {
+        keys: &[&str],
```

Comment:
this should probably be `&[String]` as converting a `Vec` of `String` to a slice of `&str` is quite difficult.

Reply:
Addressed